setup.py

```diff
@@ -145,6 +145,7 @@
     "starlette",
     "tensorflow-cpu>=2.3",
     "tensorflow>=2.3",
+    "tensorflow-probability",
```

Comment:
I'm a bit split on adding a new dependency just for one model, even if developed by the TensorFlow team. Is it really necessary for this one? If so, I think we should apply the same as for PyTorch models (like torch_scatter for TAPAS for instance) and have it be an optional dependency.

Thoughts @patrickvonplaten and @LysandreJik ?

Reply:
removed `tfp`